src/Peachpie.AspNetCore.Web/SynchronizedTextWriter.cs

```diff
@@ -81,13 +85,21 @@ public override void Write(char[] chars, int index, int count)
 
         public override void Write(char value)
         {
+#if NETSTANDARD2_0
+            var chars = new char[1] { value };
```

Comment:
I'd recommend to pre-allocate this into an instance field.

Reply:
Alright, but is there any advantages here in keeping a difference between 2.0 and 2.1? Is `Span<char>` that great for a single char?

Reply:
2.1 gets rid of most of the allocations, Span gets allocated on stack (no allocation on heap, no garbage collection), Encoding strings does not allocate either.